capnpy/segment/builder.pyx

```diff
@@ -12,6 +11,12 @@ cdef extern from "Python.h":
     int PyByteArray_Resize(object o, Py_ssize_t len)
     char* PyByteArray_AS_STRING(object o)
 
+cdef extern from "_util.h":
```

Comment:
maybe we could add a `_util.pxd`, to avoid repeating the same declarations again and again